browser/ui/brave_browser_command_controller.cc

```diff
@@ -383,6 +399,13 @@ bool BraveBrowserCommandController::ExecuteBraveCommandWithDisposition(
     case IDC_TOGGLE_JAVASCRIPT:
       brave::ToggleJavascriptEnabled(&*browser_);
       break;
+    case IDC_SHOW_PLAYLIST_BUBBLE:
+#if BUILDFLAG(ENABLE_PLAYLIST_WEBUI)
+      brave::ShowPlaylistBubble(&*browser_);
+#else
+      NOTREACHED() << " This command shouldn't be enabled";
```

Comment:
Note: I think this can be reached in weird circumstances:
1. User downloads a version of Brave with playlist enabled
2. User adds a shortcut for showing the bubble
3. User updates to a version of brave without playlist enabled
4. User presses shortcut

Probably not a very likely one though :laughing:

Reply:
I hope that could be filtered as disabled command ☺️